src/compiler/types.ts

```diff
@@ -5845,6 +5845,7 @@ namespace ts {
         include?: string[];
         exclude?: string[];
         [option: string]: string[] | boolean | undefined;
+        inferTypings?: boolean;

```

Comment:
It seems like this is only used to control whether or not ATA is done based on file names, but types are "inferred" from imports as well. Might be worth to more explicitly include the file name aspect in the setting name?